### PR TITLE
Giving the possibility to go to the comments section by the comment icons and placing for the tags to appear in the profile even when there is no

### PR DESCRIPTION
**Pull request by perospirone**
this commit depends on https://github.com/lianetoolkit/liane-toolkit/pull/195

### CODE REVIEW

File: imports/ui2/components/PagePaging.jsx [hunk 1]
```diff
@@ -105,6 +108,12 @@ class PagePaging extends Component {
           </p>
         )}
         <div className="spacer" />
+        <div className="children">
+          <button className="btn-pagination" onClick={() => {setLimit(20)}}>20</button>
```
Comment (miguelpeixe) [inline imports/ui2/components/PagePaging.jsx:112]: The UI is not clear on the purpose of these buttons. I don't think it's necessary to allow the user to change this limit.

File: package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "@mapbox/leaflet-pip": "^1.1.0",
     "@sentry/node": "^5.21.1",
     "axios": "^0.21.1",
+    "bcrypt": "^5.0.1",
```
Comment (miguelpeixe) [inline package.json:19]: We don't need to address this dependency on this PR.

**Review by miguelpeixe — CHANGES_REQUESTED** [review]
The links are working well!

Left a few requests below. The `yarn.lock` and `package.json` files should be reverted for this merge.

For reference, here's how I'm seeing my People directory page with the new navigation options:

Captura de Tela 2021-11-11 às 21 50 41

File: imports/ui2/components/PagePaging.jsx (hunk 1 above)
Comment (danitw) [reply imports/ui2/components/PagePaging.jsx:112]: but they told me to do it, it was on my trial
about the ui: I asked for dart and he said it was good

Comment (miguelpeixe) [reply imports/ui2/components/PagePaging.jsx:112]: This implementation is being applied to every view that uses the `PagePaging` component. This causes the Comments page to have buttons that doesn't trigger anything.

Comment (eldardo) [reply imports/ui2/components/PagePaging.jsx:112]: Hi @danitw , the solution was good enough for the approval of the trial, but that does not necessarily imply direct integration into production. In addition, there were 3 different solutions with different quality results, mixed in the same pull request. I think we can close this one, and then check case by case the user stories, and before create other pull requests for each one.

**Pull request closed without merging**